### DISCUSSION TITLE
[Bug] traceId cannot transfor for dubbo-spring-boot-starter:2.7.14 on apache-skywalking-apm-es7-8.5.0

### DISCUSSION THREAD

**Discussion by baymax55** [apache/skywalking#8269] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

when i use dubbo-spring-boot-starter:2.7.4.1 on apache-skywalking-apm-es7-8.5.0,i can see my consumer and provider in Trace module ,picture  as follows:
![image](https://user-images.githubusercontent.com/35788491/145320057-114d4d65-97cf-42c6-87ab-c228f47fc8ee.png)

but when i replace dubbo-spring-boot-starter version to 2.7.14, ,the picture  as follows:
![image](https://user-images.githubusercontent.com/35788491/145320168-179a284d-2295-45f4-ba56-124a2dc7704e.png)

### What you expected to happen

i hope it can work on dubbo-spring-boot-starter:2.7.14

### How to reproduce

install apache-skywalking-apm-es7-8.5.0 on service,use this [module code](https://github.com/YunaiV/SpringBoot-Labs/tree/master/lab-39/lab-39-skywalking-dubbo), replace dubbo version to 2.7.14

### Anything else

skywalking version:apache-skywalking-apm-es7-8.5.0.tar.gz       
 <dependency>
            <groupId>org.apache.dubbo</groupId>
            <artifactId>dubbo</artifactId>
            <version>2.7.14</version>
        </dependency>
        <dependency>
            <groupId>org.apache.dubbo</groupId>
            <artifactId>dubbo-spring-boot-starter</artifactId>
            <version>2.7.14</version>
        </dependency>

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I think you asked at the wrong place. Our plugin has no change for Dubbo for a long time. Dubbo3 is only known WIP. 
So, you may be better asking Dubbo community, what is changed?
